### PR TITLE
MRCD8-243 rm sub nav styles

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Added sub-navigation styles for hover menu molecule
- Removed original couple of rules from remove_me file

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the subnav looks good now that there is no JS associated with it and down to three levels are exposed
3. Compare to subnav mock at https://invis.io/DCEGPUZS4#/263924862_News_Node_Display

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-243

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: themes/math_research_center/scss/molecules/hover-menu.scss [hunk 1]
```diff
@@ -129,7 +130,79 @@
   }
 }
 
-// Styles for no javascript
+// Subnavigation - no JavaScript
+// No JavaScript is applied to the menu when
+// it does not appear in the Header region.
+.js .decanter-width-one-fourth {
```
Comment (pookmish) [inline themes/math_research_center/scss/molecules/hover-menu.scss]: I dont think we would want to target this incase we change the the width of that sidebar.

**Review by pookmish — CHANGES_REQUESTED** [review]
I can't click on the menu items on the pages with a views form below the menu. such as `/events`

File: themes/math_research_center/scss/molecules/hover-menu.scss (hunk 1 above)
Comment (josephgknox) [reply themes/math_research_center/scss/molecules/hover-menu.scss]: For sure! Was trying to find the best unique identifier. Now that we have removed the JS from every region other than the header, perhaps I replace that with .main-container.

**Comment by josephgknox** [comment]
Thanks @pookmish!

Replaced .decanter-width-one-fourth target with the region instead (.main-container). Since we're using the same menu, this is likely the best separation of the two, similar to how we are attaching the JS only to the #header region.

**Comment by josephgknox** [comment]
I am now able to click the menu. The responsive fix (removing the float and changing the display of list items seems to have addressed this for me.

What I now see: https://cl.ly/0p0V0D2Z0J2B

**Comment by pookmish** [comment]
One more thought. don't we need a hover indicator on the block title link?

**Pull request merged**